jilutil/auto_sys_job.py

```diff
@@ -118,10 +120,14 @@ def from_str(cls, jil: str):
             try:
                 # get the attribute:value pair
                 attribute, value = line.split(':', 1)
-                job[attribute.strip()] = value.strip()
+                if attribute == NOTIFICATION_EMAILADDRESS:
+                    email_address_list.append(value)
+                else:
+                    job[attribute.strip()] = value.strip()
```

Comment:
```suggestion
                if attribute in job and isinstance(job[attribute], list):
                    job[attribute].append(value)
                elif attribute in job:  # and is not list
                    job[attribute] = [job[attribute]].append(value)
                else:
                    job[attribute.strip()] = value.strip()
```
Also consider - we could have support for any repeated var like this ^ and remove the `EMAILADDRESS` specific stuff

Reply:
worthwhile to add a small test, either way

Reply:
Sounds good! Added the implementation for that + some doctests.